renovate.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "reviewers": [
+    "team:gotify"
```

Comment:
I think the team is called `committers` https://github.com/orgs/gotify/teams/committers. Can probably be omitted, because we are the default reviews via CODEOWNERS